panel/tests/ui/test_auth.py

```diff
@@ -17,8 +17,6 @@
 )
 
 pytestmark = pytest.mark.ui
-auth_check = pytest.mark.skipif('PANEL_TEST_AUTH' not in os.environ, reason='PANEL_TEST_AUTH environment variable is required to run this test')
```

Comment:
I would still keep this. To not accidentally run tests on personal systems that have the more generic environment variable set.